### ISSUE TITLE
Feature request: List pubkey along with ipv6 when running the peers command

### ISSUE THREAD

**Issue by prurigro** [inhies/cjdcmd#27]
This way the peers command can be used to cross check working peers in your config :)

**Comment by ryansb** [comment]
I don't think that makes sense for default behavior, but I could see there being a "-k" or "--keys" option.
